pkg/api/meta/priority.go

```diff
@@ -157,12 +158,19 @@ func (m PriorityRESTMapper) RESTMapping(gk unversioned.GroupKind, versions ...st
 	if err != nil {
 		return nil, err
 	}
-
 	// any versions the user provides take priority
 	priorities := m.KindPriority
 	if len(versions) > 0 {
 		priorities = make([]unversioned.GroupVersionKind, 0, len(m.KindPriority)+len(versions))
 		for _, version := range versions {
+			if strings.Count(version, "/") == 0 {
```

Comment:
> @deads2k @liggitt I want to know why ParseGroupVersion use empty group when version is not group/version format, I think it should respect groupKind not use empty group.

`versions` should be just the version part.  Group should come from the `GroupKind`.